magento/magento2
# prototype.js: check element has parent node before remove.

### Description (*)

Before call to `removeChild` on parent node, check that parent node is not null.
If the parent node is null, no action needed.

### Related Pull Requests


### Fixed Issues (if relevant)

1. Fixes magento/magento2#35387

### Manual testing scenarios (*)

1. Browse to Magento 2.4 website.
2. Open console
3. Run `document.createElement('div').remove()`
4. Expected result: nothing happen.

### Questions or comments



### Contribution checklist (*)
